js/parser.js

```diff
@@ -121,6 +121,7 @@
 
         // <sort>/<page>
         // sort and paging
+        var  pathWithCatalog = '#' + this._catalog + '/' + this._path;
```

Comment:
Remove this and just pass the path whenever you're using `pathWithCatalog`. Chaise should create the url.